packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -89,6 +92,12 @@ - (instancetype)initWithFrame:(CGRect)frame
   return self;
 }
 
+- (void)dealloc {
+  if (_historyDelegate != nil) {
+    [_historyDelegate stopObserving:_webView];
```

Comment:
Do we need to `nil` the `_historyDelegate` here? Can you please check other plugins.

Reply:
I think we don't need to because it will always be created during init